Lib/ensurepip/__init__.py

```diff
@@ -197,7 +167,8 @@ def _bootstrap(*, root=None, upgrade=False, user=False,
         if verbosity:
             args += ["-" + "v" * verbosity]
 
-        return _run_pip([*args, *_PACKAGE_NAMES], additional_paths)
+        return _run_pip([*args, "pip"], [filename])
```

Comment:
```suggestion
        return _run_pip([*args, "pip"], [str(filename)])
```

Reply:
Should that not be `os.fsdecode(filename)` instead?

Reply:
Perhaps? I considered using ``os.fspath``, but ``Path`` just calls ``str`` for ``__fspath__`` internally so I took the shortcut. I'm not sure on ``fsdecode`` vs ``fspath`` though.

Reply:
Why decode and not encode, then? Don't we pass this over the OS API boundary into another process?

Reply:
Implemented as https://github.com/python/cpython/commit/b2a69a121a82d5abc8b99eed2492c4602d9fb91b. Marking as resolved.